fem/fe.cpp

```diff
@@ -40,22 +40,19 @@ FiniteElement::FiniteElement(int D, Geometry::Type G, int Do, int O, int F)
 void FiniteElement::CalcVShape (
    const IntegrationPoint &ip, DenseMatrix &shape) const
 {
-   mfem_error ("FiniteElement::CalcVShape (ip, ...)\n"
-               "   is not implemented for this class!");
+   MFEM_ABORT("method is not implemented for this class");
```

Comment:
Is there a reason for these cases (and below)?

Reply:
I'm not sure I understand. The point of these changes is to use the macro MFEM_ABORT, which should be standard. The "old" way with `mfem_error` is pretty weird: you need to manually enter the function name, you cannot use `<<` to format the message, etc. It's true these changes are unrelated to the PR, but I tend to fix these things when stumble upon them.

Reply:
Yes, you're right — I hadn't noticed that difference between `mfem_error` and `MFEM_ABORT`.